include/RED4ext/Relocation.hpp

```diff
@@ -1,6 +1,12 @@
 #pragma once
 
 #include <cstdint>
+#include <filesystem>
+#include <string_view>
+
+#include <Windows.h>
```

Comment:
do you use WIN32_LEAN_AND_MEAN btw as a global define?

i bet this enlarges compiled objects and prolongs compile times if you don't and include it like this in bunch of places.

Reply:
Apparently we don't define this, I was pretty sure we did.